src/testSetup/groovy/com/bmuschko/gradle/docker/TestConfiguration.groovy

```diff
@@ -2,11 +2,16 @@ package com.bmuschko.gradle.docker
 
 final class TestConfiguration {
     private static final String DOCKER_SERVER_URL_SYS_PROP = 'dockerServerUrl'
+    private static final String DOCKER_HOST_SYS_PROP = 'dockerHost'
     private static final String DOCKER_CERT_PATH_SYS_PROP = 'dockerCertPath'
     private static final String DOCKER_PRIVATE_REGISTRY_URL_SYS_PROP = 'dockerPrivateRegistryUrl'
 
     private TestConfiguration() {}
 
+    static String getDockerHost() {
+        System.properties[DOCKER_HOST_SYS_PROP] ?: 'tcp://localhost:2375'
```

Comment:
If we are now using DOCKER_HOST_SYS_PROP do also still need DOCKER_SERVER_URL_SYS_PROP?